app/views/bikes/bike_fields/_revised_colors.html.haml

```diff
@@ -1,38 +1,43 @@
 - classname ||= ''
 - bike ||= @bike
-- primary_color_help_content = "The color of the frame and fork&mdash;not the wheels, cranks, or anything else. You can put a more detailed description in paint description (once you've registered), <em>this</em> is to get a general color to make searching easier"
+
+- primary_color_help_content = t(".primary_color_help_content_html")
```

Comment:
nit: since this is now a translation, rather than inline string, it doesn't need to be defined separately.

Honestly, I just did this because I didn't know how to do multiline haml tags 😅